news/_posts/2023-02-25-release-20230225.md

```diff
@@ -0,0 +1,26 @@
+---
+kind: article
+title: "Release 20230225"
+author: "PunkPun"
+created_at: 2023-02-25 00:00 +0000
+disqus_id: "release-20230225"
+permalink: "/news/release-20230225/"
+---
+
+Today we have a special occasion on our hands. We have been waiting for this moment for almost two years: A new stable OpenRA [Release 20230225]({{ '/download' | relative_url }}) is out now!
+
+<figure>
+  <video autoplay loop muted>
+    <source src="{{ '/images/news/20230225-2023.mp4' | relative_url }}" type="video/mp4">
+    <source src="{{ '/images/news/20230225-2023.webm' | relative_url }}" type="video/webm">
+    <img src="{{ '/images/news/20230225-2023.png' | relative_url }}"/>
+  </video>
+</figure>
+
+This release contains several major new features as well as many small-scale polish improvements. For more details see the previous playtest news posts [[one]({{ '/news/playtest-20221203/' | relative_url }}), [two]({{ '/news/playtest-20221223/' | relative_url }}), [three]({{ '/news/playtest-20230110/' | relative_url }}), [four]({{ '/news/playtest-20230218/' | relative_url }})] and, as always, for a full overview - [the full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/).
+
+But wait, there is more! This release brings major behind-the-scenes changes towards supporting the C&C Remastered Collection assets. As a cherry on top, we are today also releasing a development preview of ["Tiberian Dawn HD"](https://github.com/OpenRA/TiberianDawnHD/releases/tag/release-20230225)! This preview is multiplayer-compatible with the main 20230225 release, but performance, memory usage and loading times have not been optimized, and do not reflect the expected final requirements. The C&C Remastered Collection must be installed through Steam or Origin, and the project README provides [installation instructions for macOS and Linux](https://github.com/OpenRA/TiberianDawnHD#asset-installation).
+
+Incase you were wondering that we ran out of treats, worry not! Modders will also be feasting sweet this time. There were major updates to [documentation](https://docs.openra.net/en/release), all tooling received updates and [OpenRA (Mini)YAML Language Extension](https://marketplace.visualstudio.com/items?itemName=openra.oraide-vscode) has joined the group.
```

Comment:
```suggestion
In case you were wondering if we ran out of treats, worry not! Modders will also be feasting sweet this time. There were major updates to [our documentation](https://docs.openra.net/en/release), all tooling received updates and [the OpenRA (Mini)YAML Language Extension](https://marketplace.visualstudio.com/items?itemName=openra.oraide-vscode) and [the OpenRA Lua API extension](https://marketplace.visualstudio.com/items?itemName=openra.vscode-openra-lua) have joined the group.
```

Reply:
the lua extension existed for ages no?

Reply:
The first version was less than an yeah and a half ago, slightly before oraide and neither has been mentioned on the website until now.

Reply:
That can't be right. I thought for sure I had the lua extention like 5 years ago